server/test/Makefile

```diff
@@ -1,19 +1,30 @@
 CC = gcc
-CFLAGS = -std=c99 -Wall -pedantic -g
-DEPS = ../../lib/linkedlist/linkedlist.o ../../lib/hashtable/hashtable.o ../../lib/murmurhash/murmurhash.o ../../lib/marshal/marshal.o ../gwkv_ht_wrapper.o ../handle_operation.o
-OBJ = test-ht-wrapper
+CFLAGS = -pthread
+DEPS = ../../lib/marshal/marshal.o ../../lib/linkedlist/linkedlist.o ../../lib/hashtable/hashtable.o ../../lib/murmurhash/murmurhash.o ../gwkv_ht_wrapper.o ../server-kv.o ../handle_operation.o
+TARGET = test-ht-wrapper client-kv-test server-kv-test
 
 .PHONY: all test
 
-all: ht_wrapper_test
+all: $(TARGET) 
 
 rebuild: clean all
 
-ht_wrapper_test:
```

Comment:
Ah, gotcha. Can you leave the `ht-wrapper` tests separate though? It's beneficial to run `make ht_wrapper_test` to just run a subset of tests